package.json

```diff
@@ -25,11 +25,10 @@
         "@rollup/plugin-json": "^6.0.0",
         "@rollup/plugin-node-resolve": "^15.0.2",
         "@rollup/plugin-terser": "^0.4.1",
-        "css-loader": "^6.7.3",
+        "css-loader": "^7.1.2",
```

Comment:
We also should add the sample for Typescript that you created under samples perhaps ? May help other users in the future